android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIAbstractTabGroup.java

```diff
@@ -356,6 +361,9 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 			this.swipeable = TiConvert.toBoolean(newValue);
 		} else if (key.equals(TiC.PROPERTY_SMOOTH_SCROLL_ON_TAB_CLICK)) {
 			this.smoothScrollOnTabClick = TiConvert.toBoolean(newValue);
+		} else if (key.equals(TiC.PROPERTY_TABS_BACKGROUND_COLOR)
+				   || key.equals(TiC.PROPERTY_TABS_BACKGROUND_SELECTED_COLOR)) {
+			setDrawables();
```

Comment:
The `setDrawable()` call here will create views if currently null. So, we need to test if this works okay when setting these properties after the `createTabGroup()` call and before the `TabGroup.open()` call. This means the created tabs will be using the previous activity context instead of the tab group activity context.

Would you mind testing the following please?
```javascript
function createTab(title) {
	var window = Ti.UI.createWindow({ title: title });
	window.add(Ti.UI.createLabel({ text: title + " View" }));
	return Ti.UI.createTab({ title: title, window: window });
}
var tabGroup = Ti.UI.createTabGroup({
	tabs: [createTab("Tab 1"), createTab("Tab 2"), createTab("Tab 3")],
});
tabGroup.tabsBackgroundColor = "blue"; // <- Will this work?
tabGroup.tabsBackgroundSelectedColor = "red"; // <- Will this work?
tabGroup.open();
```

Reply:
The use-case here is an app that wants to to dynamically change the color of a tab. For example, to highlight a tab when new content is available or a notification has been received. An Alloy user typically has the creation properties hard-coded in XML, so, the only option is to change it dynamically in the view code. They may do this before the open call.

Reply:
I updated the PR. I will open a backport to have it ready in case no further changes are done.